<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cryto Voter v0.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/bitcoinBlue.png</normaloff>:/images/bitcoinBlue.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="bRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="bExit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lRefresh">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lMessage">
        <property name="text">
         <string>Patent Pending</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabPolls">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabActive">
       <attribute name="title">
        <string>Active Voting Question</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="6" column="0">
         <widget class="QLabel" name="lDeadline">
          <property name="text">
           <string>Estimated end date</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lBlockDeadline">
          <property name="text">
           <string>Block deadline</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="bVote">
          <property name="text">
           <string>Vote</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="lBlockDeadlineValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QGroupBox" name="boxRadioButtons">
          <property name="title">
           <string>Answers</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="vlRadio"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="8" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="lDeadlineValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lQuestion">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lTitle">
          <property name="text">
           <string>Question</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lId">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>QID</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabComplete">
       <attribute name="title">
        <string>Completed Voting Questions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtCompFilter"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lCompleteFilter">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="cCompleteColumn"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTreeView" name="tCompleteView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPending">
       <attribute name="title">
        <string>Pending Voting Questions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QComboBox" name="cPendingColumn"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtPendingFilter"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTreeView" name="tPendingView"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
